.evergreen/config_generator/components/abi_compliance_check.py

```diff
@@ -44,14 +45,24 @@ def call(cls, **kwargs):
         return cls.default_call(**kwargs)
 
 
+class TaskWithRunOn(EvgTask):
```

Comment:
Import `Task` from https://github.com/mongodb/mongo-c-driver/blob/master/.evergreen/config_generator/etc/utils.py#L13-L15 instead of defining `TaskWithRunOn`.